src/app/signin/page.tsx

```
@@ -32,167 +109,251 @@ export default function SigninPage() {
 
+              {/* Submit Button */}
               <Button
                 type="submit"
-                className="w-full bg-amber-600 hover:bg-amber-700 dark:bg-amber dark:text-white"
+                className="w-full h-11 bg-amber-600 hover:bg-amber-700 text-white font-medium shadow-lg shadow-amber-600/20 transition-all hover:shadow-xl hover:shadow-amber-600/30"
                 disabled={formik.isSubmitting}
               >
-                {formik.isSubmitting ? 'Signing In...' : 'Sign In'}
+                {formik.isSubmitting ? (
+                  <span className="flex items-center gap-2">
+                    <svg className="animate-spin h-4 w-4" viewBox="0 0 24 24" fill="none">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)